plugins/module_utils/network/iosxr/config/route_maps/route_maps.py

```diff
@@ -122,7 +123,7 @@ def generate_commands(self):
         # if state is merged, merge want onto have and then compare
         if self.state == "merged":
             wantd = dict_merge(haved, wantd)
-
+        sorted_keys = sorted(wantd.keys())
```

Comment:
sorting just the want for any specific reason?

Reply:
remove the command sorted_key one as this is needed to get the integration test correct, now made some changes into integration test rather than in config set code.